### PR TITLE
Fix/interview results

### PR DESCRIPTION
**Pull request by carmeltsadok**
Enable updating notes and questionnaire instance in interview dashboard (can see in role psy)

### CODE REVIEW

File: src/pages/InterviewDashboard/index.jsx [hunk 1]
```diff
@@ -69,20 +87,27 @@ const InterviewDashboard = () => {
...
+      .then(() => QuestionnaireService.createQuestionnaireInstance({
+          schemaId: questionnaireSchemaId,
           answers,
           eventId: event.id,
         }))
       .then(() => history.push('/'))
       .catch(() => openErrorSnackbar());
   };
 
+  const updateNote = async () => EventService.addEventNote(event.id, note);
```
Comment (PCOffline) [inline src/pages/InterviewDashboard/index.jsx:99]: Remove redundant `async`

**Review by PCOffline — APPROVED** [review]

File: src/pages/InterviewDashboard/index.jsx (hunk 1 above)
Comment (carmeltsadok) [reply src/pages/InterviewDashboard/index.jsx:99]: I want to it to be promise..

Comment (PCOffline) [reply src/pages/InterviewDashboard/index.jsx:99]: It returns the result of `addEventNode`, so it'll be a promise..

File: src/pages/InterviewDashboard/index.jsx [hunk 2]
```diff
@@ -1,15 +1,17 @@
+import { useState, useEffect } from 'react';
```
Comment (PCOffline) [inline src/pages/InterviewDashboard/index.jsx:1]: Add React import

File: src/pages/InterviewDashboard/index.jsx [hunk 3]
```diff
@@ -33,30 +38,43 @@ const InterviewDashboard = () => {
...
+          if (questionnaireSchemas?.length) {
+            const questionnaireSchema = questionnaireSchemas[0];
+            setQuestionnaireSchemaId(questionnaireSchema.id);
+            setQuestions(questionnaireSchema.questions);
+            setAnswers(Array(questionnaireSchema.questions.length).fill(undefined));
+          }
+        }).catch(() => toast(t('error.server')));
+      }
+    }
+  }, [event]);
```
Comment (PCOffline) [inline src/pages/InterviewDashboard/index.jsx:73]: Split to methods to make code more readable

File: src/pages/InterviewDashboard/index.jsx [hunk 4]
```diff
@@ -69,20 +87,27 @@ const InterviewDashboard = () => {
 
   const finishInterview = async () => {
     await EventService.addEventNote(event.id, note)
```
Comment (PCOffline) [inline src/pages/InterviewDashboard/index.jsx:89]: Remove redundant async await and simply return the promise

File: src/pages/InterviewDashboard/index.jsx [hunk 5]
```diff
@@ -109,7 +136,13 @@ const InterviewDashboard = () => {
               sm={12}
             >
               {malshab && <MalshabDetails malshab={malshab} />}
-              <Notesbox note={note} setNote={(v) => setNote(v)} bigCard={!malshab} />
+              <Notesbox
+                note={note}
+                setNote={(v) => setNote(v)}
```
Comment (PCOffline) [inline src/pages/InterviewDashboard/index.jsx:141]:
```suggestion
                setNote={setNote}
```

**Review by PCOffline — CHANGES_REQUESTED** [review]

File: src/pages/InterviewDashboard/index.jsx (hunk 5 above)
Comment (carmeltsadok) [reply src/pages/InterviewDashboard/index.jsx:141]: This is a good thing to do with setState methods

Comment (PCOffline) [reply src/pages/InterviewDashboard/index.jsx:141]: No, it's the exact same code, just your version is longer and unnecessary

File: src/pages/InterviewDashboard/index.jsx (hunk 2 above)
Comment (PCOffline) [reply src/pages/InterviewDashboard/index.jsx:1]: I meant this:
```suggestion
import React, { useState, useEffect } from 'react';
```

**Review by PCOffline — CHANGES_REQUESTED** [review]

File: src/pages/InterviewDashboard/index.jsx (hunk 2 above)
Comment (carmeltsadok) [reply src/pages/InterviewDashboard/index.jsx]: I have done it

File: mockserver/questionnaireInstance/questionnaireInstance.manager.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
+import questionnairesInstances from './db.js';
+import { generateId } from '../utils.js';
+
+class QuestionnaireManager {
+  static getInstanceById(req, res) {
+    res.send(questionnairesInstances.find((node) => node.id === req.params.id));
+  }
+
+  static async create(req, res) {
```
Comment (PCOffline) [inline mockserver/questionnaireInstance/questionnaireInstance.manager.js:9]: Remove redundant `async`

**Review by PCOffline — APPROVED** [review]

**Review by Delitefully — APPROVED** [review]

**Pull request merged**